tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -335,33 +335,22 @@ def _execute_test_suite(self, suite: TestSuiteDescription) -> bool:
         """
         suite_name = suite.name
         suite_full_name = f"{suite_name}-{self.context.image_name}"
+        suite_start_time: datetime.datetime = datetime.datetime.now()
 
         with _set_thread_name(suite_full_name):  # The thread name is added to self._log
             with set_current_thread_log(Path.home()/'logs'/f"{suite_full_name}.log"):
-                start_time: datetime.datetime = datetime.datetime.now()
```

Comment:
removed the per-test suite result and added per-test results in the 'for test in suite.tests:' loop